src/FileConventions/Library.fs

```diff
@@ -104,3 +103,37 @@ let EolAtEof(fileInfo: FileInfo) =
                 False
         else
             True
+
+let DetectInconsistentVersionsInGitHubCI(fileInfos: seq<FileInfo>) =
```

Comment:
@realmarv good work! now let's rename this func to DetectInconsistentVersionsInGitHubCIWorkflow, then create a new func DetectInconsistentVersionsInGitHubCI that receives a DirectoryInfo instead of FileInfo, which will recurively get all files underneath the directory and call DetectInconsistentVersionsInGitHubCIWorkflow with a list of all of them